### PR TITLE
LDAP: Increase profile picture limit to 512

### PR DESCRIPTION
**Pull request by marcelklehr**
* Resolves: #40267

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by kesselb — APPROVED** [review]

File: apps/user_ldap/lib/User/User.php [hunk 1]
```diff
@@ -740,7 +740,7 @@ private function setOwnCloudAvatar() {
 
 
 		//make sure it is a square and not bigger than 128x128
```
Comment (blizzz) [inline apps/user_ldap/lib/User/User.php:742]: the comment is now out of date ;)

```suggestion
		//make sure it is a square and not bigger than 512x512
```

**Review by solracsf — APPROVED** [review]

**Comment by marcelklehr** [comment]
"Computer says no." :D

**Pull request merged**